### PR TITLE
Feature/changes

### PR DESCRIPTION
**Pull request by VoronovAlexander**
Android:
- must be passed «path» saving photo
- removed «permission» responsibility
- added torch
- can disable to using gallery
- minimising noise
- changed design

iOS:
- many bugs fixed which did not allow using package
- must be passed «path» saving photo
- can disable to using gallery

Docs:
- short-example
- updated screenshots
- added info about iOS localisations from WeScan

### CODE REVIEW

**Comment by SamoSargisyan** [comment]
That's what I was looking for

**Comment by teaDrunk4rd** [comment]
Wow, that's definitely what I need. Like!

**Comment by sawankumarbundelkhandi** [comment]
thanks for the PR

File: lib/edge_detection.dart [hunk 1]
```diff
@@ -5,12 +5,31 @@ import 'package:flutter/services.dart';
...
+  static Future<bool> detectEdgeFromGallery(String saveTo,
+      {String androidCropTitle: "Crop",
+      String androidCropBlackWhiteTitle: "Black White",
+      String androidCropReset: "Reset"}) async {
+    return await _channel.invokeMethod('edge_detect_gallery', {
+      'save_to': saveTo,
+      'crop_title': androidCropTitle,
+      'crop_black_white_title': androidCropBlackWhiteTitle,
+      'crop_reset_title': androidCropReset,
+    });
```
Comment (sawankumarbundelkhandi) [inline lib/edge_detection.dart:33]: @VoronovAlexander are we not supposed to pass 
'from_gallery' : true
 for detectEdgeFromGallery to work properly?

**Pull request merged**